doc/btp_bap.txt

```diff
@@ -362,6 +362,23 @@ Commands and responses:
         In case of a success, the IUT continues processing the command
         asynchronously.
 
+    Opcode 0x1a - Scan Delegator Add Source
+
+                Controller Index:       <controller id>
+                Command parameters:     Broadcaster Address Type (1 octet)
+                                        Broadcaster Address (6 octets)
+                                        Advertiser SID (1 octet)
+                                        Broadcast ID (3 octets)
+                                        BIG Encryption (1 octet)
+                                        PA State (1 octet)
```

Comment:
Seem my comment from https://github.com/zephyrproject-rtos/zephyr/pull/99788#discussion_r2559990522 w.r.t. the order of arguments